index.adoc

```diff
@@ -602,6 +602,38 @@ MUST NOT perform type validation on any value in the array before submitting the
 
 Use of this member is OPTIONAL.
 
+A non-normative example
+
+[source,json]
+----
+{
+  "href": "https://example.io/persons", "rel": ["create-form"], "method": "POST",
+  "value": [
+    {
+      "name": "friends", "type": "object", "form": {
+        "type": "set", "itemtype": "object",
+        "value": [
+          { "name": "firstName" },
+          { "name": "email" }
+        ]
+      }
+    }
+  ]
+}
+----
```

Comment:
👍  Glad to hear it!  We still might have some minor modifications concerning `etype` and `eform` and when an Ion parser should look for them, but I agree that #30 and #31 have made things are more intuitive and warrant the new members.  Thanks for the feedback!